spec/jobs/subscription_placement_job_spec.rb

```diff
@@ -115,6 +115,18 @@
           expect(changes[line_item2.id]).to be 3
           expect(changes[line_item3.id]).to be 3
         end
+
+        context "and the order has been placed" do
+          before do
+            allow(order).to receive(:ensure_available_shipping_rates) { true }
+            allow(order).to receive(:process_each_payment) { true }
+            job.send(:place_order, order.reload)
+          end
+
+          it "removes the unavailable items from the shipment" do
+            expect(order.shipment.manifest.size).to eq 1
```

Comment:
A little pro-tip to spec this more precisely:

```suggestion
            expect {
              job.send(:place_order, order.reload)
            }.to change {
              order.shipment.manifest.size
            }.from(2).to(1)
```

But I don't want to delay this PR further.

Reply:
Oh nice, thanks for the tip @mkllnk! That's really nice and expressive. I tried it out and in this specific case the shipment doesn't actually exist before calling `place_order`, so I think it's best to leave it as is.